pkgs/development/python-modules/lintrunner/default.nix

```diff
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  maturin,
+  rustPlatform,
+  cargo,
+  rustc,
+  python,
+}:
+
+buildPythonPackage rec {
+  pname = "lintrunner";
+  version = "0.12.7";
+  format = "pyproject";
+
+  # NOTE: there is a PR open on the upstream repository for the Cargo.lock file.
+  # After that is merged we can change the owner and rev to the upstream
+  # repository instead of a fork.
+  src = fetchFromGitHub {
+    owner = "PrestonHager";
+    repo = pname;
+    rev = "main";
+    #rev = "v${version}";
+    hash = "sha256-CYSNEDT9RO7bnVW8zcDDJOva4GOKGMNs1CcU//IbBrc=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit src;
+    hash = "sha256-dv/CznOB/VFx1PM+cYmdHuiK7d8z/HUQbDt5iy5LUTI=";
+  };
+
+  # Adding maturninBuildHook will automatically build and install the executable
+  nativeBuildInputs = [
+    maturin
+    rustPlatform.cargoSetupHook
+    rustPlatform.maturinBuildHook
+    cargo
+    rustc
+  ];
+
+  meta = with lib; {
+    description = "A tool that runs linters.";
+    longDescription = ''
+      Lintrunner is a tool that runs linters by deciding which files need to be
+      linted, invoking linters according to a common protocol, and gathering
+      results and presenting them to users.
+    '';
+    homepage = "https://github.com/suo/lintrunner";
+    changelog = "https://github.com/suo/lintrunner/releases/tag/v${version}";
+    mainProgram = "lintrunner";
+    license = licenses.mit;
```

Comment:
dual license?
https://github.com/suo/lintrunner/blob/1b70da01a6a9237c0f60b07d017f6fb60f49ac8c/Cargo.toml#L7

Reply:
I left an issue in the upstream repository, [#84](https://github.com/suo/lintrunner/issues/84). If I were to guess, it's meant to be MIT and the other one is from Cargo doing something. In the meantime, is there a dual license option we can do to specify both licenses for the nix package?

Reply:
Didn't realize that license could be a list. So added the second license to the package definition.